### ISSUE TITLE
Parts are missing in the encoder part

### ISSUE THREAD

**Issue by TientJan** [drinkingcoder/FlowFormer-Official#27]
Hello, in the `MemoryEncoder Class`, it seems that there should be added the definition of `self.layers`. In the `if self.cfg.feat_cross_attn` part of `forward()` function, `self.layers` is used without definition.

**Comment by qinliuliuqin** [comment]
Hi, this issue has been mentioned in #23.

**Comment by drinkingcoder** [comment]
Yes. This option should always be False. We will trim our code that is out of date.
